### ISSUE TITLE
Don't use jokes for genuine instructions

### ISSUE THREAD

**Issue by Naln1** [airtonix/avahi-aliases#11]
TL;DR: Using a joke for genuine instructions makes life more difficult on people who don't recognise the joke, English is not their first language, and neurodivergent people who are not prepared to receive a joke in what should be genuine help.

![Naln1: ...plz don't use jokes for genuine instructions (Image of Creating Aliases section of Readme.MD) iz just blegh... 
rainflake: ya, is unclear 
Naln1: especially for people who don't know the joke, or english is not their first language
rainflake: yup
Naln1: or ND people whose parsers are set up for instructions and not jokes
rainflake: yus
Naln1: i was that third group for a few seconds / "???"? What does "???" mean!? DIDN'T YOU WRITE THE BLOODY PROGRAM!?! HOW DO YOU NOT KNOW WHAT TO DO HERE? / _reads step 5 for context_ / Profit? Wot? You can't... make money with this in any wa- oh it's... it's the stupid joke from Reddit... 
rainflake: i didnt notice profit for a while x3 got stuck on ???
Naln1: ^SEE](https://user-images.githubusercontent.com/43188089/129413876-8da3ab8f-7551-48eb-acbd-2f1d6659f6bc.png)
